tests/unit/amo/components/TestCollection.js

```diff
@@ -107,6 +107,12 @@ describe(__filename, () => {
     expect(wrapper.find(AddonsCard)).toHaveProp('loading', true);
   });
 
+  it('renders placeholder text if there are no add-ons', () => {
+    const wrapper = renderComponent({ collection: null });
+
+    expect(wrapper.find('.Collection-placeholder')).toHaveLength(1);
+  });
+
```

Comment:
We should also have a test that the placeholder does not appear when there are add-ons in the collection.